src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-wrapper.service.ts

```diff
@@ -50,8 +50,8 @@ export class TaskWrapperService {
   _handleExecutingTasks(task: FinishedTask) {
     this.notificationService.show(
       NotificationType.info,
-      this.taskManagerMessageService.getRunningMessage(task),
-      this.taskManagerMessageService.getDescription(task)
+      this.taskMessageService.getRunningTitle(task),
+      this.taskMessageService.getErrorTitle(task)
```

Comment:
This should be removed.
We don't have any description to show when the task is running.